source/dub/dub.d

```diff
@@ -1631,7 +1631,7 @@ private class DependencyVersionResolver : DependencyResolver!(Dependency, Depend
 	protected override Dependency[] getSpecificConfigs(string pack, TreeNodes nodes)
 	{
 		if (!nodes.configs.path.empty || !nodes.configs.repository.empty) {
-			if (getPackage(pack, nodes.configs)) return [nodes.configs];
+			if (getPackage(nodes.pack, nodes.configs)) return [nodes.configs];
```

Comment:
Duh, glad to see it was a dumb typo in the end.

Reply:
yeah, no idea how the hashOf removal broke this though, can only guess that it was missing checks further inside, because path based dependencies used to always equal to true whenever comparing with any other path based dependencies.